tests/test_support.py

```diff
@@ -296,16 +297,55 @@ def raise_attribute_error():
         assert str(exception.value) == 'message'
 
 
-WHEN = datetime.datetime(2007, 4, 1, 15, 30)
-
-def test_format_datetime(timezone_getter):
-    fmt = support.Format('en_US', tzinfo=timezone_getter('US/Eastern'))
-    assert fmt.datetime(WHEN) == 'Apr 1, 2007, 11:30:00\u202fAM'
-
-
-def test_format_time(timezone_getter):
-    fmt = support.Format('en_US', tzinfo=timezone_getter('US/Eastern'))
-    assert fmt.time(WHEN) == '11:30:00\u202fAM'
+class TestFormat:
+    def test_format_datetime(self, timezone_getter):
+        when = datetime.datetime(2007, 4, 1, 15, 30)
+        fmt = support.Format('en_US', tzinfo=timezone_getter('US/Eastern'))
+        assert fmt.datetime(when) == 'Apr 1, 2007, 11:30:00\u202fAM'
+
+    def test_format_time(self, timezone_getter):
+        when = datetime.datetime(2007, 4, 1, 15, 30)
+        fmt = support.Format('en_US', tzinfo=timezone_getter('US/Eastern'))
+        assert fmt.time(when) == '11:30:00\u202fAM'
```

Comment:
Could you revert this (and unwrap the new tests from the class?), unless there's some reason for the class I'm not quite seeing here? 😁

Reply:
For me it seemed logical and also felt like following the convention in this test module.

`class LazyProxy` is tested in `class LazyProxyTestCase`
`class NullTranslations` is tested in `class NullTranslationsTestCase`
`class Translations` is tested in `class TranslationsTestCase`

So it would be logical to have tests for `class Format` also grouped into once class. Though following the convention it should have been named to `FormatTestCase`.

Should I unwrap or change the class name?

Reply:
Though changing the name isn't probably good idea as pytest doesn't recognize `FormatTestCase` unless it inherits `unittest.TestCase` which on the other hand doesn't find `timezone_getter` fixture in `test_format_datetime` test function. 

So either unwrapping or keeping as it is now.

Reply:
It's okay. I'm just trying to get rid of `unittest.TestCase`s where they don't make sense (where they're just there for legacy reasons).